src/pages/signup/index.tsx

```diff
@@ -43,8 +43,8 @@ const schema: JSONSchemaType<User> = {
   },
 };
 
-function sendVerificationEmail(user: User) {
-  emailjs.send(
+async function sendVerificationEmail(user: User,navigate: NavigateFunction) {
```

Comment:
![image](https://github.com/HelloIAmBanana/bank-hafifa/assets/33515941/14d05ac2-0214-4a50-a05c-1a62045dcfac)

I found it more readable and elegant .